src/test/groovy/org/prebid/server/functional/tests/BidValidationSpec.groovy

```diff
@@ -212,4 +212,34 @@ class BidValidationSpec extends BaseSpec {
         def metrics = defaultPbsService.sendCollectedMetricsRequest()
         assert metrics["adapter.generic.requests.bid_validation"] == initialMetricValue + 1
     }
+
+    def "PBS should return error when between refers to unknown bidder alias"() {
```

Comment:
Misleading test naming. What does `between` means? Looks like test should be named `PBS should return an error when GVL Id alias refers to unknown bidder alias`.

Reply:
Same, that comment wasn't answered here nor resolved in the code.